Ports/acpica-tools/patches/0001--Stop-compiler-warnings-on-dangling-pointer.patch

```diff
@@ -0,0 +1,25 @@
+From bf6a3c66f9f4a536ad7bab3eb7f149c7062de851 Mon Sep 17 00:00:00 2001
+From: Liav A <liavalb@gmail.com>
+Date: Fri, 27 May 2022 10:18:35 +0300
+Subject: [PATCH 1/2] Stop compiler warnings on dangling pointer
+
+---
+ source/components/utilities/utdebug.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/source/components/utilities/utdebug.c b/source/components/utilities/utdebug.c
+index f789e68..fd18f5c 100644
+--- a/source/components/utilities/utdebug.c
++++ b/source/components/utilities/utdebug.c
+@@ -182,7 +182,7 @@ void
+ AcpiUtInitStackPtrTrace (
+     void)
+ {
+-    ACPI_SIZE               CurrentSp;
++    static ACPI_SIZE               CurrentSp;
```

Comment:
This can't be used for getting the stack pointer anymore though... (the `static` CurrentSp now lives in .bss).

Not sure if this breaks anything. If it does, you can use `__builtin_frame_address`.